pkg/apis/resource/types.go

```diff
@@ -139,6 +139,57 @@ type ResourceSliceSpec struct {
 	// +optional
 	// +listType=atomic
 	Devices []Device
+
+	// perDeviceNodeSelection defines whether the access from nodes to
+	// resources in the pool is set on the ResourceSlice level or on each
+	// device. If it is set to true, every device defined the ResourceSlice
```

Comment:
nits:

Don't we usually use the Go field name here not the yaml name (IOW, capitalize "per" to "Per"; same with comments below)? Also:

s/device/Device/
s/defined/defined in/

Reply:
We actually use the actual field name in API doc, here is a umbrella issue we worked earlier to clean up the existing API documentation: https://github.com/kubernetes/kubernetes/issues/113438

Reply:
Let's at least be consistent inside the same doc block :grin: 

Below it uses Go names ("Exactly one of NodeName, NodeSelector, AllNodes, and PerDeviceNodeSelection..."), then it should be the same at the start of the block.

Reply:
The rest of the package also uses Go field names.

Reply:
Updated to use GO field here to keep it consistent within the same API group. :)